### PR TITLE
Add OS support for Aviat WTM

### PR DESCRIPTION
**Pull request by joshuabaird**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/OS/AviatWtm.php [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	 * @return array
+   */
+
+  public function discoverWirelessFrequency()
+  {
+      $carrier1_oid = '.1.3.6.1.4.1.2509.9.5.2.1.1.1.59';
+      $carrier2_oid = '.1.3.6.1.4.1.2509.9.5.2.1.1.1.60';
+      # The SNMP agent on WTM devices will choke on 15+ rapid requests.
+      # A "sleep" helps LibreNMS cope with this current issue
+      sleep(5);
```
Comment (murrant) [inline LibreNMS/OS/AviatWtm.php:56]: These sleep can have a very detrimental effect on LibreNMS.  You'll need another solution.
Try setting nobulk or snmp_max_oid in the os def.

File: tests/snmpsim/aviat-wtm.snmprec [hunk 2]
```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|
```
Comment (murrant) [inline tests/snmpsim/aviat-wtm.snmprec:1]: You must have run collect-snmp-data.php before adding your sensors, because there is no data here to test.

**Comment by PipoCanaja** [comment]
Hi @joshuabaird 
Could you please rename the MIB files with their "official" name, removing the extension. The name of the mib is the text before `DEFINITIONS ::= BEGIN`

**STXN-GLOBALREGISTER-MIB** `DEFINITIONS ::= BEGIN`

Thnx

**Comment by joshuabaird** [comment]
@murrant Thanks for helping get this over the finish line!  Sorry, I've been busy and haven't been able to spend a lot of time on it.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-65-release-changelog-june-2020/12687/1

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-65-release-changelog-june-2020/12687/1

**Comment by joanrierapol** [comment]
Does this OS work with WTM4800 Multi-band? The radio does not detected... could it be adapted? Thanks!!

**Comment by joshuabaird** [comment]
Aviat may have made some changes that require some updates.  However, the support for Aviat WTM radios still doesn't work very well due to the fact that the Aviat SNMP agent on these radios can't handle more than 10-15 consecutive requests.  Aviat Engineering is aware of this, and hopefully we'll see a fix sometime in the future.

**Pull request merged**